tests/utils/tools.php

```diff
@@ -448,24 +456,24 @@ function START($id, array $options = array()) {
         ),
     );
     $ctx = stream_context_create($opts);
-    $json = file_get_contents(getMOUri() . "/servers/$id", false, $ctx);
+    $json = file_get_contents(getMongoOrchestrationURI() . "/servers/$id", false, $ctx);
     $result = json_decode($json, true);
 
     /* Failed -- or was already started */
     if (!isset($result["mongodb_uri"])) {
-        DELETE($id);
-        define($id, false);
+        destroyTemporaryMongoInstance($id);
+        die("Could not start temporary server instance\n");
```

Comment:
Can we throw an exception instead? The error handlers in `basic-skipif.inc` will ensure that any uncaught exception gets properly turned into an `exit` statement if run during a SKIPIF. If called as part of a normal test body, I also think an exception is more consistent with what we do elsewhere (avoiding utility functions that simply abort the PHP process on their own).